.github/workflows/validate-pr.yml

```diff
@@ -0,0 +1,77 @@
+name: Validate Pull Request
+
+on:
+  pull_request:
+    branches:
+      - develop
+      - main
+
+env:
+  NODE_OPTIONS: '--max-old-space-size=4096'
+  CHOKIDAR_USEPOLLING: 1
+
+jobs:
+  build:
+    name: Gatsby Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 12
+
+      - name: Cache dependencies
+        id: yarn-cache
+        uses: actions/cache@v2
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-node-modules-${{ hashFiles('**/yarn.lock') }}
+
+      - name: Cache Gatsby build
+        uses: actions/cache@v2
+        with:
+          path: |
+            public
+            .cache
+          key: ${{ runner.os }}-gatsby-build-${{ github.run_id }}
+          restore-keys: |
+            ${{ runner.os }}-gatsby-build-
+
+      - name: Install dependencies
+        if: steps.yarn-cache.outputs.cache-hit != 'true'
+        run: yarn install --frozen-lockfile
+
+      - name: Gatsby build
+        run: yarn build:production --log-pages
+        env:
+          GATSBY_EXPERIMENTAL_PAGE_BUILD_ON_DATA_CHANGES: true
+          CI: true
+
+  lint:
```

Comment:
Can we also run tests as part of PR validation?

Reply:
a bunch of the tests are failing (which is why I excluded it) because I didn't know if we wanted to address that first. For example, there's tests for our old footer

Reply:
Ah thats good to know!

I'd say lets still add it, just not make it a required check for now. It will be a really good reminder to fix when we open up future PRs.

Reply:
Been added in!